index.bs

```diff
@@ -187,6 +186,108 @@ If the console is not open when the printer operation is called,
 implementations should buffer messages to show them in the future up to an
 implementation-chosen limit (typically on the order of at least 100).
 
+<h4 id="loglevel-severity">Indicating <var>logLevel</var> Severity</h4>
+
+This section groups together {{console}} methods whose <var>logLevel</var>s produce output
+indicating similar levels of severity.
+
+<div class="example" id="loglevel-example">
+  Since <code>log</code> and <code>debug</code> produce similarly formatted outputs, they are
+  grouped together by their log level severity in the following table.
+</div>
+
+<table>
+  <thead>
+    <tr>
+      <th>{{console}} method</th>
+      <th>Log Level Severity</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>log, trace, dir, dirxml, group, groupCollapsed, debug</code></td>
+    <td>log</td>
+    <td>
+      Console message with implementation-specific formatting designed to indicate a generic log
+    </td>
+  </tr>
+  <tr>
+    <td><code>count, info, timeEnd</code></td>
+    <td>info</td>
+    <td>
+      Console message with implementation-specific formatting designed to make the message stand
+      out as informative
+    </td>
+  </tr>
+  <tr>
+    <td><code>warn</code></td>
+    <td>warn</td>
+    <td>
+      Console message with implementation-specific formatting designed to warn the user of
+      something indicated by the message
+    </td>
+  </tr>
+  <tr>
+    <td><code>error, assert</code></td>
+    <td>error</td>
+    <td>
+      Console message with implementation-specific formatting designed to indicate an error to the
+      user
+    </td>
+  </tr>
+</table>
+
+While the above table groups together {{console}} methods by <var>logLevel</var> severity, the
+Printer is still implementation-defined, and is therefore allowed to provide formatting variations
+on a per-method basis while remaining compliant. Some common examples of this UX improvisation are
+as follows:
+
+<ul>
+  <li>
+    <code>timeEnd</code> custom formatting.
+    <div class="example" id="timeEnd-formatting">
+      Formatting applied to output produced by calls to <code>timeEnd</code> may be unique to this
+      method.
```

Comment:
While expanding this example I found that Chrome 58 gives this:

![latest](https://cloud.githubusercontent.com/assets/9669289/25878101/24b5ad6e-34f6-11e7-937b-eaf39c2d83c1.png)

...basically the `info` formatting and its variants have disappeared. Do you think we should reconsider the spec and possible get rid of the `info` log level, or just include a picture of another implementation and maybe file a bug (though I'm not sure if that'll is technically necessary) on Chrome?

I tried to checkout the source to understand what was going on or when this change was made. Both
 - https://cs.chromium.org/chromium/src/v8/src/inspector/v8-console-message.cc?type=cs&l=391
 - https://cs.chromium.org/chromium/src/v8/include/v8.h?type=cs&l=6804

indicate that an internal log level of `info` still exists, among others...but I'm really not sure what is going on past that.